### DISCUSSION TITLE
Line breaks stop working on Android app

### DISCUSSION THREAD

**Discussion by paperfairy** [standardnotes/forum#2888] [General]
App v3.45.4 (3002288)
Device is Pixel 5a
![image](https://user-images.githubusercontent.com/3393711/199286138-9568ea6f-3f39-40a9-9473-5109bbf136e9.png)

A few weeks ago, I enabled the new mobile interface to try it out. Ever since enabling it, line breaks will stop working after the app has been open for longer than 10 seconds. Pressing return just immediately also enters backspace.

This does not happen on any other app on my phone. 

I can also no longer go back to the old mobile interface. Attached is a video of the bug, and as well as a screenshot of my device information.

Here is a video of the issue: https://photos.app.goo.gl/ULWdieitTG16yUXr8

**Comment by effieeee** [comment]
Hi @paperfairy, can replicate this :( Quite strange since this was already fixed in the past. Will have to revisit this 🙏

**Reply by paperfairy** [reply]
1. Why was this changed to a conversation topic? Feels very clearly like a bug.
2. Is there a fix for this? It's been six months where I can't use the app on my phone. :(
